custom/icds/management/commands/migrate_pse_attendance.py

```diff
@@ -12,10 +12,10 @@
 UPDATE {new_table} B
```

Comment:
I think these tables will need quotes around them because tehy have - in the name